packages/components/src/RadioGroup/RadioGroup.css

```diff
@@ -39,9 +39,15 @@
 
 .input[disabled] + .label {
   color: var(--color-blue--lighter);
+  cursor: not-allowed;
 }
 
 .input[disabled] + .label::before {
   border-color: var(--color-grey--light);
   background-color: var(--color-grey--lighter);
 }
+
+.description {
+  margin-bottom: var(--space-small);
```

Comment:
This looks good, can we do the same to `Checkbox` description?

![image](https://user-images.githubusercontent.com/39704901/99627116-e83af580-29f0-11eb-9100-bd5dda72c73d.png)

![image](https://user-images.githubusercontent.com/39704901/99627048-c80b3680-29f0-11eb-9aa3-667bedefcd9a.png)